exquisite_corpus/preprocess.py

```diff
@@ -7,10 +7,19 @@
 from .reddit_ban_data import BANNED_SUBREDDITS
 
 
+# This regex matches Twitter handles starting with @
 TWITTER_HANDLE_RE = regex.compile(r"@[\S--\p{punct}]+")
+
+# This regex matches Twitter URLs, which are always shortened with t.co
 TCO_RE = regex.compile("http(?:s)?://t.co/[a-zA-Z0-9]+")
```

Comment:
I fixed an issue of `.` not being escaped in an upcoming PR, but I also noticed that URLs with underscores in them will not be matched by this. Perhaps that's never an issue? I'm not sure how Twitter URLs work.